docs/components/component.md

```diff
@@ -114,6 +114,36 @@ In the example above, the following allows `todo-list` to display the provided a
 
 For more information on configuring assets, please see Stencil's [Assets Guide](../guides/assets.md)
 
+
+### `formAssociated`
```

Comment:
Minor, can we remove these backticks? We don't use them on any other properties for the decorator, and they're causing some styling differences:
![Screenshot 2023-10-09 at 3 14 11 PM](https://github.com/ionic-team/stencil-site/assets/1930213/300660c2-eb7d-4fb8-93d6-e562289d6b39)

In the image above, the red circles are meant to show the difference in the background & font of these headers. The blue calls out the same for the sidebar

Reply:
sure, although I'd maybe argue they should all have them! haha